src/stores/plugins/api.ts

```diff
@@ -51,7 +63,7 @@ if (process.env.NODE_ENV !== 'production') {
     return config;
   });
 
-  axios.interceptors.response.use((response: AxiosResponse) => {
+  axiosInstance.interceptors.response.use((response: AxiosResponse) => {
```

Comment:
todo: the interceptors can be removed, since we now use the MockServiceWorker, which gives better feedback about executed requests.

<img width="765" alt="grafik" src="https://user-images.githubusercontent.com/10571945/214262199-819440d7-2a0e-4c7b-ae5c-fd8bf6cd3cb6.png">